roles/kubernetes-apps/external_cloud_controller/openstack/tasks/main.yml

```
@@ -3,9 +3,30 @@
   include_tasks: openstack-credential-check.yml
   tags: external-openstack
 
+- name: External OpenStack Cloud Controller | Test if external_openstack_cacert is a base64 string
+  set_fact:
+    external_openstack_cacert_is_base64: "{% if external_openstack_cacert is search('^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{3}= | [A-Za-z0-9+/]{2}==)?$') %}true{% else %}false{% endif %}"
```

This should be 
```suggestion
    external_openstack_cacert_is_base64: "{{ external_openstack_cacert is search('^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{3}= | [A-Za-z0-9+/]{2}==)?$') }}"
```

And also, don't put that in a fact, use `vars:` in the task where it is relevant.